front/src/components/map/KakaoMap.vue

```diff
@@ -36,15 +36,9 @@ export default {
   },
   async mounted() {
     const map = await this.$kakaoMap.drawMap(this.$refs.map);
+    await this.giveCustomizedMessage();
     this.$store.commit("map/SET_MAP", map);
-    await this.$kakaoMap
-      .setCenterByCurrentLocation()
-      .catch(() =>
-        this.$store.commit(
-          "snackbar/SHOW",
-          ERROR_MESSAGE.UNIDENTIFIABLE_LOCATION,
-        ),
-      );
+    await this.$kakaoMap.setCenterByCurrentLocation().catch(() => {});
```

Comment:
```suggestion
    await this.$kakaoMap.setCenterByCurrentLocation();
```
catch가 비워질거라면 그냥 없어도 괜찮지 않을까요?

Reply:
빈 catch 문이어도 없으면 동작을 안 하네요! 에러 메세지는 밑에서 띄워주니까 일단은 놔두겠습니다 🙂